src/setup.jl

```diff
@@ -14,7 +14,7 @@ function Load()
 
     # Server setup
 
-    global ms = MaximaSession()	# Spin up a Maxima session
+    global ms = MaximaSession(ArgString="")	# Spin up a Maxima session
```

Comment:
Again should be `MaximaSession(ArgString=ArgString)` to pass the keyword through